smoke_tests/sqs_send_message_batch_test.go

```diff
@@ -370,7 +372,7 @@ func TestSendMessageBatchV1_Xml_Success_including_attributes(t *testing.T) {
 		WithFormField("Entries.1.MessageBody", messageBody2).
 		WithFormField("Entries.1.MessageAttributes.1.Name", binaryAttributeKey).
 		WithFormField("Entries.1.MessageAttributes.1.Value.DataType", binaryType).
-		WithFormField("Entries.1.MessageAttributes.1.Value.BinaryValue", binaryValue).
+		WithFormField("Entries.1.MessageAttributes.1.Value.BinaryValue", binaryValueEncodeString).
```

Comment:
Here is also for [blobs are sent and received after being base64 encoded](https://smithy.io/2.0/aws/protocols/aws-query-protocol.html).
BinaryValue in XML should be base 64 encoded.